src/Sylius/Bundle/AdminApiBundle/Resources/config/services.xml

```diff
@@ -23,6 +23,8 @@
         <defaults public="true" />
 
         <service id="Sylius\Bundle\AdminApiBundle\Command\CreateClientCommand">
+            <argument on-invalid="null"/>
```

Comment:
It still passes an empty string as command name which causes `InvalidArgumentException` to be thrown. What was wrong with `<argument>null</argument>`?